alert-policies/nextcloud/ScrapeErrors.yml

```diff
@@ -0,0 +1,37 @@
+name: Scrape Errors
+
+description: |+
+  This alert is triggered  when the cache size limit is reached above 4 for atleast 5 minutes.
```

Comment:
"This alert is triggered when the cache size limit is reached above 4 for at least 5 minutes."